src/lib/paypay-rest-sdk.ts

```diff
@@ -15,28 +15,30 @@ export interface HttpsClientMessage {
 class PayPayRestSDK {
   private options: any = "";
   private productionMode: boolean = false;
+  private perfMode: boolean = false;
```

Comment:
Why do you keep instance variables?

Reply:
@tilfin we are refactoring the whole SDK (repeating config files etc), we will be pushing that feature branch in a day or two please feel free to contribute on that draft PR.